docs/framework/whats-new/index.md

```diff
@@ -1165,9 +1165,9 @@ For example, the following XAML groups data by age, sort the age groups in ascen
 
 **Soft keyboard support**
 
-Soft Keyboard support enables focus tracking in a WPF applications by automatically invoking and dismissing the new Soft Keyboard in Windows 10 when the touch input is received by a control that can take textual input.
+Soft Keyboard support enables focus tracking in WPF applications by automatically invoking and dismissing the new Soft Keyboard in Windows 10 when the touch input is received by a control that can take textual input.
```

Comment:
Apparently we're not supposed to use soft keyboard: https://styleguides.azurewebsites.net/StyleGuide/Read?id=2547&topicid=14398